docs/how-to/customize-branding.mdx

```diff
@@ -9,18 +9,44 @@ keywords:
 pagination_next: null
 pagination_prev: null
 last_update: 
-   date: 06/01/2023
-   author: Patricia McPhee
+   date: 08/13/2023
+   author: Harry Guo
 draft: false
 doc_type: how-to
 displayed_sidebar: mainSidebar
 ---
 
-import { Alert, Col, Row } from 'antd';
+## Introduction
 
-<Row>
-  <Col span={6}>
-    <Alert message="To do" type="warning" />
-  </Col>
-</Row>
-<br />
\ No newline at end of file
+In the Beyond Identity Console, there are a couple of places where you can customize the branding to fit your organization's needs. 
+The branding configuration is **realm-specific** and we've broken it down to the sections below.
+
+### General
```

Comment:
I think you can probably make all of these sections with 2 #'s, otherwise it looks like they are nested under Introduction
```suggestion
## General
```